lib/attr_encrypted/adapters/active_record.rb

```diff
@@ -43,6 +43,12 @@ def assign_attributes(*args)
             def attributes=(*args)
               perform_attribute_assignment :attributes_without_attr_encrypted=, *args
             end
+
+            alias_method :attributes_without_attr_encrypted, :attributes
+            def attributes
+              encrypted_keys = self.class.encrypted_attributes.keys
+              attributes_without_attr_encrypted.reject { |k, _| encrypted_keys.include?(k.to_sym) }
+            end
```

Comment:
I guess there is an inconsistency. `#has_attribute?` will return `true` for encrypted attributes as it relies on `@attributes`.